components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIConsumer.java

```diff
@@ -319,7 +319,7 @@ public void addComment(APIIdentifier identifier, String comment,
      * @param userId      User Id
      * @throws APIManagementException if failed to add Application
      */
-    public String addApplication(Application application, String userId) throws APIManagementException;
+    public int addApplication(Application application, String userId) throws APIManagementException;
```

Comment:
JAVA docs doesn't explain the return value of the method. What is the int value? what possible values can it take? and what are meaning of those?